src/obu.rs

```diff
@@ -124,7 +126,10 @@ impl Debug {
     }
 }
 
-fn parse_seq_hdr(c: &mut Rav1dContext, gb: &mut GetBits) -> Rav1dResult<Rav1dSequenceHeader> {
+fn parse_seq_hdr(
```

Comment:
```suggestion
#[inline(never)]
fn parse_seq_hdr(
```

Porting C's `NOINLINE` addition.